src/services/url.ts

```diff
@@ -146,7 +147,8 @@ class UrlService extends BaseService<UrlService.Options> {
   }
 
   static detailsUrlState(state: Store.State): UrlBeautifier.DetailsUrlState {
-    const details = Selectors.details(state);
+    const details: Store.Product =
+      <any>ProductTransformer.transform(Selectors.details(state).data, Selectors.config(state)['structure']);
```

Comment:
It won't let met cast it to a product so I just declared it any and made the variable a product.